x/claim/client/cli/query_claim_record.go

```diff
@@ -0,0 +1,46 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/arkeonetwork/arkeo/x/claim/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+)
+
+var _ = strconv.Itoa(0)
+
+func CmdClaimRecord() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "claim-record [address]",
+		Short: "Query claim-record",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			reqAddress := args[0]
+
```

Comment:
consider to validate the address, make sure it is legit before sending it to the server

Reply:
yes, this actually needs to be fixed further since a user will need to also pass the chain.

Reply:
[7c3008b](https://github.com/arkeonetwork/arkeo/pull/29/commits/7c3008b2207bcac40095d1d330407ad61975ff6b)